src/App.js

```diff
@@ -11,6 +14,22 @@ import {  useState } from "react";
  */
 function App() {
   const [selectedQuery, setSelectedQuery] = useState(undefined)
+  const session = getDefaultSession()
+  const [loggedIn, setLoggedIn] = useState()
+  useEffect(() => {
+    session.onLogin(() => setLoggedIn(true))
+    session.onLogout(() => setLoggedIn(false))
+
+    handleIncomingRedirect({restorePreviousSession: true})
+      .then((info) => {
+        if(info){
+          let status = info.isLoggedIn || false
+          if(status !== loggedIn){
```

Comment:
What is happening here?

Reply:
Not sure why I put the or false there but it is certainly unnecessary I have removed the line in  [commit 726d991](https://github.com/SolidLabResearch/generic-data-viewer/pull/23/commits/726d991e059130a52fb35dbd64289f4421309b89)